Makefile.common

```diff
@@ -41,6 +41,12 @@ ifneq "$(TIDB_EDITION)" "Enterprise"
 endif
 endif
 
+ENTERPRISE_EXTENSION_VERSION :=
```

Comment:
I think it is `ENTERPRISE_EXTENSION_GIT_HASH` in fact ?

Reply:
Yes, `ENTERPRISE_EXTENSION_GIT_HASH` is more exact. Let me correct it.